app/src/main/java/com/framgia/forder/screen/shopupdate/ShopUpdateViewModel.java

```diff
@@ -29,4 +33,57 @@ public void onStop() {
     public void setPresenter(ShopUpdateContract.Presenter presenter) {
         mPresenter = presenter;
     }
+
+    public String getShopName() {
+        if (mShopManagement.getShop() != null) {
+            return mShopManagement.getShop().getName();
+        }
+        return "";
+    }
+
+    public String getShopDescription() {
+        if (mShopManagement.getShop() != null) {
+            mShopManagement.getShop().getDescription();
+        }
+        return "";
+    }
+
+    public String getTimeAutoClose() {
+        if (mShopManagement.getShop() != null) {
+            mShopManagement.getShop().getTimeAutoClose();
+        }
+        return "";
+    }
+
+    public String getTimeAutoReject() {
+        if (mShopManagement.getShop() != null) {
+            mShopManagement.getShop().getTimeAutoReject();
```

Comment:
ERROR: [STYLE][RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT] Return value of com.framgia.forder.data.model.Shop.getTimeAutoReject() ignored, but method has no side effect
http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT